### PR TITLE
navigation changes for account freeze and account switching

### PR DESCRIPTION
**Pull request by nitheesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/4854

*Description of changes:*

*Checklist:*
I confirm that below checklist items are addressed in this pull request; (check the boxes applicable)
- [ ] No lint errors
- [ ] No test case failures and proper coverage for all modules/classes
- [ ] Updated the deployment configs for new environment variable(s)
- [ ] Updtaed the postman collection in entity repository (https://github.com/bcgov/entity/tree/master/api-e2e/postman) for e2e tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: auth-web/src/components/auth/mixins/NextPageMixin.vue [hunk 1]
```diff
@@ -159,10 +157,19 @@ export default class NextPageMixin extends Vue {
       // eslint-disable-next-line no-console
       console.log('Redirecting user to Account Freeze message since the account is temporarly suspended.')
       if (this.currentMembership?.membershipTypeCode === MembershipType.Admin) {
```
Comment (saravanpa-aot) [inline auth-web/src/components/auth/mixins/NextPageMixin.vue:159]: can this Admin check be removed and replaced with permission check?

File: auth-web/src/routes/router.ts [hunk 2]
```diff
@@ -134,7 +135,12 @@ export function getRoutes (): RouteConfig[] {
       path: '/account/:orgId/settings',
       name: 'account-settings',
       component: accountSettings,
-      meta: { requiresAuth: true, requiresProfile: true, requiresActiveAccount: true },
+      meta: {
+        requiresAuth: true,
+        requiresProfile: true,
+        requiresActiveAccount: true,
+        allowOnAccountFreeze: true
```
Comment (saravanpa-aot) [inline auth-web/src/routes/router.ts:142]: allowOnAccountFreeze - any way we can tie this with permissions? making it backend driven?

**Review by saravanpa-aot — CHANGES_REQUESTED** [review]

File: auth-web/src/routes/router.ts (hunk 2 above)
Comment (nitheesh-aot) [reply auth-web/src/routes/router.ts:142]: we can make a mapper in the frontend with the route names and permission values, but it still be a dependency with frontend when we add new permission or route changes.
which will be a small effort, but can get ignored sometimes.
if its ok, I can work like that

**Review by saravanpa-aot — APPROVED** [review]

File: auth-web/src/routes/router.ts (hunk 2 above)
Comment (nitheesh-aot) [reply auth-web/src/routes/router.ts:142]: https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5743

**Pull request merged**